scripts/package-build/build.py

```
@@ -136,9 +136,26 @@ def build_package(package: list, patch_dir: Path) -> None:
+            package_version = package['commit_id'].replace('/', '_')
+
+        package_name = run(['dpkg-parsechangelog', '--show-field', 'Source', '--file', repo_dir / 'debian/changelog'], capture_output=True, check=False)
+        if package_name.stdout:
+            package_name = package_name.stdout.decode().strip()
+        else:
+            # On failure fallback to repo name
+            package_name = repo_name
+
+        # Build a tarball for the package
+        tarball_name = f'{package_name}_{package_version}.orig.tar.gz'
         run(['tar', '--exclude=.git', '--exclude=.github', '-czf', tarball_name, '-C', str(repo_dir.parent), repo_name], check=True)
```

Ruff (0.15.15)

[error] `subprocess` call: check for execution of untrusted input

(S603)